.github/workflows/build.yml

```diff
@@ -87,11 +87,15 @@ jobs:
       - name: Install test dependencies
         run: |
           . ../firedrake_venv/bin/activate
+          cd ../firedrake_venv/src/pytest-mpi
+          git checkout JDBetteridge/spawn
+          cd -
           python "$(which firedrake-clean)"
           python -m pip install \
-            pytest-xdist pytest-timeout ipympl
+            pytest-xdist pytest-timeout ipympl pytest-split
+          pip install git+https://github.com/JDBetteridge/mpispawn
```

Comment:
Before we merge this we should probably put this into the firedrakeproject organisation, or pin to a version or something?